app/src/main/java/com/door43/translationstudio/core/TargetTranslationMigrator.java

```diff
@@ -121,6 +126,8 @@ private static File v5(File path) throws Exception {
             id += "_" + resourceSlug;
         }
 
+        addLicenseFile(path);
```

Comment:
This is the only place this is added so there's no need to place this in a method.